src/App.js

```diff
@@ -1,19 +1,42 @@
 import React from 'react';
 
-import './App.scss';
+import './App.css';
+import { Tabs } from './components/Tabs';
 
 /* eslint-disable */
 const tabs = [
-  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+  { id: 'tab-1', title: 'Home', content: 'Some text 1' },
+  { id: 'tab-2', title: 'About', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Contacts', content: 'Some text 3' },
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+class App extends React.Component {
+  state = {
+    newTabs: [...tabs],
```

Comment:
make it easy, remove `new`
```suggestion
    tabs: [...tabs],
```